src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/RadioLabel/strings.ts

```diff
@@ -15,50 +15,29 @@ interface RadioLabelStrings {
-    nonContextualizedNotRecommended:
-      "Not recommended for epidemiological interpretation due to lack of visibility into viral diversity outside of the selections below and omission of closely-related samples.",
-    overviewBestFor:
-      "Best for generating a summary tree of samples of interest, in the context of genetically similar GISAID samples.",
-    overviewDescription:
-      "Builds tree focused around samples of interest (defined by location, date, and lineage) and closely-related samples, at a ratio of roughly 2:1.",
-    overviewGoodFor1: "Good for identifying possible local outbreaks.",
-    overviewGoodFor2:
-      "Good for specifying sample location, collection date, or lineages instead of the defaults used for your CZ GEN EPI weekly build.",
-    targetedBestFor: "Best for investigating an identified outbreak.",
-    targetedDescription:
-      "Builds tree with selected samples (from the sample table or the ID box below) and closely-related samples, at a ratio of roughly 1:2.",
-    targetedGoodFor:
-      "Good for identifying samples most closely related to the selected samples among all samples in GISAID and your CZ GEN EPI samples.",
-  },
-  // TODO: (ehoops) - these are just copy pasta to appease typescript.  Copy changes have their own ticket.
-  [Pathogen.MONKEY_POX]: {
-    nonContextualizedBestFor:
-      "Best for uncovering sampling bias in your own sampling effort.",
-    nonContextualizedDescription:
-      "Builds tree with only samples of interest (defined by location, date, and lineage).",
-    nonContextualizedGoodFor:
-      "Good for seeing viral diversity in the public repository that may not be captured by your own sampling effort.",
-    nonContextualizedNotRecommended:
-      "Not recommended for epidemiological interpretation due to lack of visibility into viral diversity outside of the selections below and omission of closely-related samples.",
-    overviewBestFor:
-      "Best for generating a summary tree of samples of interest, in the context of genetically similar GISAID samples.",
-    overviewDescription:
-      "Builds tree focused around samples of interest (defined by location, date, and lineage) and closely-related samples, at a ratio of roughly 2:1.",
-    overviewGoodFor1: "Good for identifying possible local outbreaks.",
-    overviewGoodFor2:
-      "Good for specifying sample location, collection date, or lineages instead of the defaults used for your CZ GEN EPI weekly build.",
-    targetedBestFor: "Best for investigating an identified outbreak.",
-    targetedDescription:
-      "Builds tree with selected samples (from the sample table or the ID box below) and closely-related samples, at a ratio of roughly 1:2.",
-    targetedGoodFor:
-      "Good for identifying samples most closely related to the selected samples among all samples in GISAID and your CZ GEN EPI samples.",
-  },
+  [Pathogen.COVID]: generalViralRadioLabelStrings,
```

Comment:
I'm wondering if you think it's worth having a `default` case or something for `PathogenConfigType`, so that a pathogen's strings only need to be specified when it differs from the default?

I'm on the fence about the idea, though. May make more sense to wait until we have 3+ pathogens because then default will actually mean something other than "are they both the same?".

Reply:
Definitely worth considering.  For a few different cases I created a similar generalViral string object and used that for mpox (and was planning to use for hepC as well). Either this or a default would potentially work with a spread operator and then just changing the specific strings that need it.  

I agree that we should revisit once we have 3+ pathogens.